llvm/lib/Frontend/Offloading/OffloadWrapper.cpp

```diff
@@ -186,57 +186,60 @@ GlobalVariable *createBinDesc(Module &M, ArrayRef<ArrayRef<char>> Bufs,
                             ".omp_offloading.descriptor" + Suffix);
 }
 
-void createRegisterFunction(Module &M, GlobalVariable *BinDesc,
-                            StringRef Suffix) {
+Function *createUnregisterFunction(Module &M, GlobalVariable *BinDesc,
```

Comment:
It seems the order of createRegisterFunction and createUnregisterFunction is swapped. This causes some artificial differences. Is it OK to keep their original order.

Reply:
No, since I need to call this function from `createRegisterFunction` now. I could forward declare it but I don't think there's a point given it's inside an anonymous namespace.